tests/lib/rules/useSelector-prefer-selectors.js

```diff
@@ -0,0 +1,75 @@
+require('babel-eslint');
+
+const rule = require('../../../lib/rules/useSelector-prefer-selectors');
+const RuleTester = require('eslint').RuleTester;
+
+const parserOptions = {
+  ecmaVersion: 6,
+  sourceType: 'module',
+  ecmaFeatures: {
+    experimentalObjectRestSpread: true,
+  },
+};
+
+const ruleTester = new RuleTester({ parserOptions });
+
+ruleTester.run('useSelector-prefer-selectors', rule, {
+  valid: [
+    'const property = useSelector(xSelector)',
+    {
+      code: 'const property = useSelector(xSelector)',
+      options: [{
+        matching: '^.*Selector$',
+      }],
+    },
+    {
+      code: 'const property = useSelector(getX)',
+      options: [{
+        matching: '^get.*$',
+      }],
+    },
+    {
+      code: 'const property = useSelector(selector)',
+      options: [{
+        matching: '^selector$',
+      }],
+    },
+  ],
+  invalid: [{
+    code: 'const property = useSelector((state) => state.x)',
+    errors: [
+      {
+        message: 'useSelector should use a selector function.',
```

Comment:
@martini97  Thanks a lot for the PR. 

Would it be possible to rephrase this error. I believe `(state) => state.x` is a function technically.
I guess it is not a named function it is anonymous function.  Not sure if there is a better term to describe this error.